bundles/org.openhab.binding.webexteams/src/main/resources/OH-INF/i18n/webexteams.properties

```diff
@@ -0,0 +1,37 @@
+# binding
+binding.webexteams.name = Webex Teams Binding
+binding.webexteams.description = Send messages to Webex Teams using actions
+
+# thing types
+thing-type.webexteams.account.label = Webex Teams Account
+thing-type.webexteams.account.description = Webex Teams Account
```

Comment:
I see this is not exactly the same texts as in the XML files.
Please rather use "mvn i18n:generate-default-translations".

Reply:
DOne.  Thanks for the suggestion.